addon/components/file-uploader/component.js

```diff
@@ -90,10 +91,7 @@ export default Ember.Component.extend({
 
   _upload() {
     let uploader = EmberUploader.Uploader.create({
-      url: this.get('url'),
-      method: this.get('method'),
-      paramName: this.get('attribute').underscore(),
```

Comment:
Are you sure this will work for our current use case ? Some file are create from scratch, some others update an existing model like  https://github.com/upfluence/publishr-supplier-web/blob/ae06b527e19cf3aac12b5135daa2578c6f0e8b13/app/payment-request/us/supporting-document/template.hbs#L30

Reply:
this idea is push back the update logic back to the domain. So basically didUpload will return an artifact (with a key) and from that we should just mess around with that key in the ember app and just:

```
didUpload(artifact) {
  model.set('supportingDocuemnt', artifact.key);
  model.save();
}
```

Reply:
Yep of course, it's to be sure this was take in account. To not forget it in the next bump of ember-upf-utils.